ansible/roles/test/files/ptftests/IP_decap_test.py

```diff
@@ -178,18 +186,30 @@ def create_encap_packet(self, dst_ip, outer_pkt='ipv4', triple_encap=False):
         tc_in = tos_in = dscp_in << 2
         dscp_out = random.randint(0, 32)
```

Comment:
Probably it's a good idea to remove this rand value also

Reply:
I have commited a fix which picks DSCP and TTL values from allowed range in a round robin way.